src/k-provider/ott/provider-parser.js

```diff
@@ -23,7 +23,7 @@ const MediaTypeCombinations: {[mediaType: string]: Object} = {
     [KalturaPlaybackContext.Type.TRAILER]: () => ({type: MediaEntry.Type.VOD}),
     [KalturaPlaybackContext.Type.PLAYBACK]: mediaAssetData => {
       if (parseInt(mediaAssetData.externalIds) > 0 || mediaAssetData.objectType === LIVE_ASST_OBJECT_TYPE) {
-        return {type: MediaEntry.Type.LIVE, dvrStatus: 0};
+        return {type: MediaEntry.Type.LIVE, dvrStatus: mediaAssetData.enableTrickPlay ? 1 : 0};
```

Comment:
the condition in the ticket discuss `enableTrickPlay` and `mediaAssetData.objectType === LIVE_ASST_OBJECT_TYPE` but not `mediaAssetData.externalIds`

Reply:
Correct. pushed a fix

Reply:
Can we add a test?

Reply:
We can, OTT doesn't have at all provider parser spec. I think we should add. Just a matter of time :)

Reply:
please discuss with @yairans to add unit tests coverage for this